templates/baremetalset/cloudinit/networkdata

```diff
@@ -0,0 +1,11 @@
+links:
+- name: {{ .MgmtInterface }}
+  id: {{ .MgmtInterface }}
+  type: vif
+networks:
+- netmask: {{ .MgmtNetmask }}
+  link: {{ .MgmtInterface }}
+  id: {{ .MgmtInterface }}
+  ip_address: {{ .MgmtIp }}
+  type: ipv4
+  gateway: {{ .MgmtGateway }}
```

Comment:
ditto here on gateway. we may need to make it configurable and possible unset by default for the management network.